python/tests/test_cyr_nar_to_cyr_taras.py

```diff
@@ -0,0 +1,22 @@
+import json
+import pytest
+import glob
+
+from translit import convert
+from translit.cases import Case
+
+
+def read_json_data():
+    test_file_list = glob.glob('../tests/test_*.json')
+    all_test_cases = []
```

Comment:
How about making it a dict from filename to list of tests? Or alternatively marking each test with filename that it came from? During test failure it would be good not only print the test name, but also the filename it came from for easier debugging.

Reply:
Fixed <img width="1022" alt="image" src="https://github.com/belaudiobooks/lacinka/assets/8971400/145bd254-f98c-4bd6-af24-7cfc328cec3a">